inlong-manager/manager-web/src/main/resources/application-prod.properties

```diff
@@ -44,24 +41,31 @@ spring.datasource.druid.testOnReturn=false
 spring.datasource.druid.filters=stat,wall
 # Open the mergeSql function through the connectProperties property, Slow SQL records
 spring.datasource.druid.connectionProperties=druid.stat.mergeSql=true;druid.stat.slowSqlMillis=5000
-
 # Manager address of Tube cluster, used to create Topic
 cluster.tube.manager=http://127.0.0.1:8081
 # Master address, used to manage Tube broker
 cluster.tube.master=127.0.0.1:8000,127.0.0.1:8010
 # Tube cluster ID
 cluster.tube.clusterId=1
-
 # Push configuration to the path on ZooKeeper
 cluster.zk.url=127.0.0.1:2181
 cluster.zk.root=inlong_hive
-
 # Application name in Sort
 sort.appName=inlong_app
-
 # Pulsar admin URL
 pulsar.adminUrl=http://127.0.0.1:8080,127.0.0.2:8080,127.0.0.3:8080
 # Pulsar broker address
 pulsar.serviceUrl=pulsar://127.0.0.1:6650,127.0.0.1:6650,127.0.0.1:6650
 # Default tenant of Pulsar
 pulsar.defaultTenant=public
+# Elasticsearch config
```

Comment:
ditto

Reply:
It has done

Reply:
It has done by add config item,The following are the specific configuration item: Audit query source that decide what data source to query, currently only supports [MYSQL|ELASTICSEARCH]